sycl/doc/extensions/GroupMask/GroupMask.asciidoc

```diff
@@ -178,64 +176,57 @@ work-item with the id `max_local_range()-1`.
 |`void flip(id<1> id)`
 |Toggle the value of the bit corresponding to the specified _id_.
 
-|`bool operator==(group_mask rhs) const`
+|`bool operator==(const group_mask &rhs) const`
```

Comment:
Why change all these to take `const&`?

Reply:
make reference to avoid copying opblect. Make reference const to avoid modifications of rhs object.